src/utils/server-utils.ts

```diff
@@ -504,13 +504,17 @@ export async function fetchWMSLayerAsGeoJSON(options: {
       );
     }
 
-    const wfs = new WFS(lyr.baseUrl);
+    const wfsServerURL = `${lyr.baseUrl}/wfs`;
+
+    const wfs = new WFS(wfsServerURL);
 
     const wfsLayer = await wfs.getLayer(lyr.serverLayerName);
 
     const featureCollection: GeoJSON.FeatureCollection = await wfsLayer.getFeatures(
       {
+        count: Infinity,
```

Comment:
Is there a risk in setting this to Infinity? What would be a reasonable finite value otherwise?

Reply:
Hey, good question.  I'll set it to 100,000.  If we run into a situation where we are using a WFS service that has tens of thousands of features (within a given time frame), we might want to consider implementing paging to improve performance and network reliability.

Reply:
done